src/main/resources/application.properties

```diff
@@ -9,12 +9,19 @@ spring.rabbitmq.host=${BROKER_HOST:mq}
 spring.rabbitmq.port=${BROKER_PORT:5672}
 
 inbox.port=${INBOX_PORT:2222}
-inbox.directory=${INBOX_LOCATION:/ega/inbox/}
 inbox.cache.ttl=${CACHE_TTL:3600}
 
+inbox.local.directory=${INBOX_LOCATION:/ega/inbox/}
+
+inbox.s3.endpoint=${S3_ENDPOINT:inbox-backend:9000}
+inbox.s3.region=${S3_REGION:us-east-1}
```

Comment:
Then we should set `lega` as a region to Minio server as well, because Minio defaults to `us-east-1}` if nothing is specified.

Reply:
And, actually, this is a different region. The one that you are referencing is for Vault S3. For Inbox S3 we don't have such default value there.

Reply:
The S3 backend will most likely be the same both for inbox and data archive.
And in a testing environment, (minio), it needs to be the same.

Reply:
That's not completely accurate, at least for Norway. We are going to have different S3's for Inbox and for Vault. The first one will be publicly exposed, but the second one will be not. So they may have different regions set up (or may have the same). Anyway, I don't see a problem here. It's just some default value, it could be anything: `ca-central-1`, `lega` or `blah-blah-blah`. You can always override it via env-var, the same you can do with Minio. But since we are using Minio at the moment, I've picked Minio's default value as default value here.